config/services.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+/** @var Silex\Application $app */
+use Baikal\Framework\Silex\ControllerResolver;
+
+$app->register(new Silex\Provider\TwigServiceProvider(), [
+    'twig.path' => $baseDir . '/views/',
+]);
+
+$app->register(new Silex\Provider\Psr7ServiceProvider());
```

Comment:
I guess we can live without PSR-7. I assumed it was desired, at least a reason for looking at Slim (#502). I have no problem removing it again. Maybe until it can add some real benefit, we can do without it.